server/manager/src/main/java/org/apache/accumulo/manager/upgrade/UpgradeCoordinator.java

```diff
@@ -142,11 +143,17 @@ public synchronized void upgradeZookeeper(ServerContext context,
         "Not currently in a suitable state to do zookeeper upgrade %s", status);
 
     try {
-      int cv = context.getServerDirs()
-          .getAccumuloPersistentVersion(context.getVolumeManager().getFirst());
-      ServerContext.ensureDataVersionCompatible(cv);
+      int cv = AccumuloDataVersion.getCurrentVersion(context);
       this.currentVersion = cv;
 
+      int oldestVersion = ROOT_TABLET_META_CHANGES;
```

Comment:
Instead of having another place where this is hard-coded, can just get the first upgrader mapping key:

```suggestion
      int oldestVersion = upgraders.iterator().next().getKey();
```

However, this requires a predictable ordering of the upgraders... which we might already be relying on, and which might be an existing problem. See my other comment on this.

Reply:
Addressed in 7c8ec3ad63